pkg/continuousprofiling/convert.go

```diff
@@ -0,0 +1,59 @@
+package continuousprofiling
+
+import "github.com/grafana/pyroscope-go"
+
+var (
+	acsToPyroscopeProfileType = map[ProfileType]pyroscope.ProfileType{
```

Comment:
Why do you introduce the `ProfileType` and `ProfilerConfiguration` abstractions? I like the option pattern, but I wonder if this is perhaps a bit unnecessary boiler plate.

Reply:
You are right. Right now there is a little bit of unnecessary boiler plate. My thought process is that if at some point we want to swap pyroscope for something else or expand on it, we have all the changes located in this package and we can hide those changes from the consumer of the package.

Reply:
Fair enough. I'll remove the conversion functions 👍